MassSpectrometry/Chromatography.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Chemistry;
+using MzLibUtil;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace MassSpectrometry
+{
+    public class Chromatography
+    {
+        /// <summary>
+        /// Extracts an ion chromatogram from a spectra file, given a mass, charge, retention time, and mass tolerance.
+        /// </summary>
+        public static ExtractedIonChromatogram ExtractIonChromatogram(MsDataFile file, double mass, int charge, Tolerance massTolerance,
+            double retentionTime, int msOrder = 1, double retentionTimeWindowWidth = 5)
```

Comment:
for times i think we need to specify units (either min or sec)

Reply:
I prefer minutes just because 4000 seconds is not as intuitive to me as 66.6 minutes but maybe that's just a personal thing. i'll rename it though